dev_guide/deployments.adoc

```diff
@@ -395,8 +397,6 @@ than one, the  first replica of the deployment will be validated for readiness
 before fully scaling up the deployment. If the validation of the first replica
 fails, the deployment will be considered a failure.
 
-IMPORTANT: When executing the "post" lifecycle hook, all failures will be
-ignored regardless of the failure policy specified on the hook.
```

Comment:
I am removing this snippet since it is no longer true (you can have whatever failure policy you want on your post hook and it will work fine.